raiden-network/light-client
# dapp: fix action buttons for channels on mobile

**Thank you for submitting this pull request :)**

Fixes #2420

**Short description**
This basically just removes some Vuetify magic we don't need but causes the issue.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Serve the dApp locally (use PR tool?)
2. Make sure you have at least one channel open in the selected token network
3. Goto the channels view
4. Make sure the buttons to withdraw and deposit work fine on desktop
5. Switch to the mobile view in the dev tools (or Ctrl+Shift+m)
6. Make sure that the buttons still look fine (note that there is no fancy slider effect on mobile as before)
